lib/kitchen/figure_element.rb

```diff
@@ -20,7 +20,25 @@ def initialize(node:, document: nil)
     # @return [Element, nil]
     #
     def caption
-      first('figcaption')
+      first('figcaption', "div[data-type='description']")
```

Comment:
Can you add to `Kitchen::Selectors::Base` and `Kitchen::Selectors::Standard1` then say 

```ruby
first(selectors.figure_caption)
```

this will allow it to be overridden later if needed

Reply:
Also, while this is test covered by `BakeFigure`'s usage of it, might be nice to have a quick spec in the `figure_element_spec.rb` to show that this captures both kinds of captions.